src/packages/media/media/collection/media-collection.element.ts

```diff
@@ -48,12 +49,29 @@ export class UmbMediaCollectionElement extends UmbCollectionDefaultElement {
 		this._progress = event.progress;
 	}
 
+	#refreshCollection() {
```

Comment:
To avoid over updating the UI this has been added to we don't update the UI more than every 350ms